docs/reference/enrich-processor/normalize-to-otel-processor.md

```diff
@@ -0,0 +1,149 @@
+---
+navigation_title: "Normalize to OTel"
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/normalize-to-otel-processor.html
+---
+
+# Normalize-to-OTel processor [normalize-to-otel-processor]
+
+
+Detects whether a document is OpenTelemetry-compliant and if not - normalizes it as described below. The resulting document can be queried seamlessly by clients that expect either [ECS](https://www.elastic.co/guide/en/ecs/current/index.html) or OpenTelemetry-[Semantic-Conventions](https://github.com/open-telemetry/semantic-conventions) formats.
```

Comment:
The resulting doc can not automatically be queried both ways, it needs the otel mapping component templates for that as well. We should either add that or leave it out.

Reply:
Of course! I will add this. I don't think it makes sense to leave it out as this is the purpose of this processor, right? This one sentence is important, it should capture the essence of what this processor is good for. 

If you want to write something for this summary - I think it would be best. Otherwise, I will propose a version that includes the OTel component templates. Please refer me to which are they.

Reply:
It's fine to start small. We should reference the `logs-otel@template` index template here.

Reply:
@flash1293 please see if the [current wording](https://github.com/elastic/elasticsearch/pull/125699/files#diff-12dc3d32727f60ba943302b2c52725b23ee9b820093713e65c997c625e482927R10-R14) is what you expect

Reply:
Looks great, thanks Eyal!